### DISCUSSION TITLE
Removing results from site assets library

### DISCUSSION THREAD

**Discussion by manjunathreddy1** [microsoft-search/pnp-modern-search#3293] [General]
Hi All,

I'm trying to remove search results from site assets library using the following query templates but it is not working. Please confirm if there is any possibility to fix this issue.

(-Path:'*/SiteAssets/*')
(-webUrl:'*/SiteAssets/*')

**Comment by wobba** [comment]
`NOT ParentLink:SiteAssets` should do the trick
